docs/dev/test-configuration-options.md

```diff
@@ -474,10 +492,15 @@ Identifies the browser to be used when automating with a mobile browser. See the
 
 <!-- prettier-ignore -->
 :::note
-- If this capability is not provided for a virtual device, the ['app'](#app) capability needs to be set. If none is set the test will throw an error.
-- This capability can be omitted for virtual devices if the ['app'](#app) capability is set.
-- If this capability is not provided for a real device session and also the ['app'](#app) capability is not provided then a real device session will automatically default back to the default browser. This will be Chrome for Android and Safari for iOS
-:::
+
+- If this capability is not provided for a virtual device, the [`appium:app`](#appiumapp) capability needs to be set. If none is set the test will throw an error.
+- This capability can be omitted for virtual devices if the [`appium:app`](#appiumapp) capability is set.
+- If this capability is not provided for a real device session and also the:
+  - [`appium:app`](#appiumapp)
+  - or [`appium:bundleId`](#appiumbundleid) (iOS)
+  - or [`appium:appPackage`](#appiumapppackage) and ['appium:appActivity'](#appiumappactivity) (Android)
+  capability are not provided, then a real device session will automatically fallback to the default browser. This will be Chrome for Android and Safari for iOS
```

Comment:
```suggestion
  capability is not provided, then a real device session will automatically fall back to the default browser. This will be Chrome for Android and Safari for iOS
```